apps/feedahelp/src/components/ui/features/fahCounter/FAHCounter.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import FAHCounterDisplay from "./FAHCounterDisplay";
+
+const FAHCounter = () => {
+  return (
+    <div className="grid grid-cols-1 gap-8 p-10 lg:grid-cols-2 xl:grid-cols-4">
+      <FAHCounterDisplay
+        title={"TOTAL MONTHLY DONATIONS ($)"}
+        imageIcon={"/static/images/feedahelp/donate.png"}
+        imageAlt={"donation-icon"}
+        value={100000}
+        monthlyIncrease={"+4.4%"}
+      />
+      <FAHCounterDisplay
```

Comment:
make a JSON data file. data will be an array of objects. each object will have title ,imageIcon, imageAlt, value, monthlyIncrease.
Import data here and map it. apply DRY (do not repeat yourself)